tests/e2e/local/kubeadm/install_prereqs_debian.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+source "../common_linux.sh"
+
+check_apt_get
+sudo apt-get --quiet -y update
+
+check_dpkg
+
+install_curl
+
+install_docker
+
+install_kubectl
+
+echo "Checking and Installing kubeadm as required"
+if ! kubeadm --help > /dev/null; then
+  curl -LO https://storage.googleapis.com/kubernetes-release/release/"$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)"/bin/linux/amd64/kubeadm
+  chmod +x ./kubeadm
+  sudo mv ./kubeadm /usr/local/bin/kubeadm
+fi
+
+echo "Checking and Installing kubelet as required"
+if ! kubeadm --help > /dev/null; then
```

Comment:
Should this be `kubelet`?

Reply:
Please note that it's just in WIP state and I will ping you when it is ready for review.

Reply:
@sublimino 

This PR is ready for review now. Please take a look again when you got chance :)